### PR TITLE
added sample Filterbuttons to landing page

### PR DESCRIPTION
**Pull request by JohnZ9865**

### CODE REVIEW

**Comment by JohnZ9865** [comment]
image
image

**Comment by JohnZ9865** [comment]
image
image

File: src/components/landing/Landing.jsx [hunk 1]
```diff
@@ -5,13 +5,17 @@ import ProjectTypes from "@/components/landing/ProjectType";
...
+import Filterbuttons from "../projects/filterbuttons";
 
 const Landing = () => {
   return (
     <div className="w-full bg-blue-50 flex flex-col">
       <Navbar />
       <Welcome />
       <About />
+
+      <Filterbuttons />
```
Comment (fardinzam) [inline src/components/landing/Landing.jsx:17]: filter buttons shouldn't be in the landing page, it should be in the projects page

File: src/components/projects/filterbuttons.jsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          </div>
+        );
+      })}
+    </div>
+  );
+}
+
+const Filterbuttons = () => {
+  const [open, setOpen] = useState(false);
+  const [open1, setOpen1] = useState(false);
```
Comment (fardinzam) [inline src/components/projects/filterbuttons.jsx:43]: give them more descriptive labels, makes it easier for others to figure out what it's for
same for the handleOpen names

**Comment by fardinzam** [comment]
> image image

the colors are a little off, use #8EC5D5 for the blue and #FFC633 for the yellow:
Screenshot 2024-01-19 at 9 38 24 AM

**Comment by JohnZ9865** [comment]
image
image
I configured the colors above in tailwind config.js.

**Pull request closed without merging**